### PR TITLE
feat: compound plugin to sync board/repo config to db

### PR DESCRIPTION
**Pull request by klesh**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing](../CONTRIBUTING.md) Documentation
- [x] This PR is using a `label` (bug, feature etc.)
- [x] My code is has necessary documentation (if appropriate)
- [ ] I have added any relevant tests
- [ ] This section (**⚠️ &nbsp;&nbsp;Pre Checklist**) will be removed when submitting PR

# Summary

### Key Points

- [ ] This is a breaking change
- [ ] New or existing documentation is updated

### Description
Describe what this PR does, and aims to solve in a few sentences.

### Does this close any open issues?
Please mention the issues here.

### Current Behavior
Describe the current behaviour of this issue, if relevant.

### New Behavior
Describe the new behaviour updated in this issue, if relevant.

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by joncodo** [comment]
since there are possibly many combined plugins, we need a better name than compound. Maybe something like jiraGithub

**Comment by klesh** [comment]
@joncodo   Yes, agree, totally.
It's not even a final design, just for Sep Demo only.
We will discuss how should we organize plugin dependency after Sep Demo.
So, let's keep the name until then.

**Comment by joncodo** [comment]
I can't merge this without a README explaining config and how to use this plugin

File: .env.example [hunk 1]
```diff
@@ -14,3 +14,7 @@ JIRA_BASIC_AUTH_ENCODED=emhl..........................................a0QzQUE=
 GITLAB_ENDPOINT=https://gitlab.com/api/v4/
 GITLAB_AUTH=xxx
 JIRA_ISSUE_EPIC_KEY_FIELD=customfield_10014
+
+
+# Compound BOARD_ID:PROJECT_ID1,PROJECT_ID2
+JIRA_BOARD_GITLAB_PROJECTS=8:8967944,8967945;9:8967946,8967947
```
Comment (joncodo) [inline .env.example:20]: What are these and how do they work?

File: plugins/compound/compound.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+type Compound string
+
+func (plugin Compound) Init() {
+	err := lakeModels.Db.AutoMigrate(
+		&models.JiraBoardGitlabProject{},
+	)
+	if err != nil {
+		panic(err)
+	}
+	text := config.V.GetString("JIRA_BOARD_GITLAB_PROJECTS")
```
Comment (joncodo) [inline plugins/compound/compound.go:23]: text seems generic. What does text mean?

File: plugins/compound/compound.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	if text == "" {
+		return
+	}
+	fmt.Printf("text: %v\n", text)
+	rows := make([]*models.JiraBoardGitlabProject, 0)
+	for _, comp := range strings.Split(text, ";") {
```
Comment (joncodo) [inline plugins/compound/compound.go:29]: Is there a way we can set an array or object in the config instead of splitting strings?

**Review by joncodo — APPROVED** [review]

**Pull request merged**